NuRadioReco/modules/RNO_G/dataProviderRNOG.py

```diff
@@ -16,10 +16,10 @@ class dataProviderRNOG:
     It does so by running a set of "processing" modules on the data.
     This module is a wrapper around the following modules (in this order):
 
-    #. `NuRadioReco.modules.io.RNO_G.readRNOGDataMattak`
-    #. `NuRadioReco.modules.RNO_G.channelGlitchDetector`
-    #. `NuRadioReco.modules.RNO_G.channelBlockOffsetFitter`
-    #. `NuRadioReco.modules.channelAddCableDelay`
+    - `NuRadioReco.modules.io.RNO_G.readRNOGDataMattak`
+    - `NuRadioReco.modules.RNO_G.channelGlitchDetector`
+    - `NuRadioReco.modules.RNO_G.channelBlockOffsetFitter`
+    - `NuRadioReco.modules.channelAddCableDelay`
```

Comment:
Why?

Reply:
This list could be rendered by my IDE, the one before not. Since I am using VSCode I assume the experience is the same for may users.